.eslintrc.js

```diff
@@ -1,5 +1,12 @@
 module.exports = require("@blitzjs/next/eslint")
 
+module.exports = {
+  extends: ["./node_modules/@blitzjs/next/eslint"],
+  rules: {
+    "import/order": "error",
+  },
+}
```

Comment:
@ohrie bist du sicher, dass das das richtige macht? Zeile 1 setzt ja den module.export und dieser Block überschreibt ihn dann, richtig?

Das müsste IMO dann eher sowas sein…, damit die Infos von Zeile 1 erhalten und ergänzt werden.

```suggestion
module.exports = {
  ...module.exports,
  extends: ["./node_modules/@blitzjs/next/eslint"],
  rules: {
    "import/order": "error",
  },
}
```

Reply:
Das habe ich noch nie gesehen. Ich glaube das wird durch extends schon sichergestellt. Siehe auch https://eslint.org/docs/latest/use/configure/configuration-files#using-eslintrecommended

Reply:
@ohrie ja, das macht Sinn.

Übrigens sieht die eslint bei Blitz-Atlas jetzt so aus https://github.com/FixMyBerlin/atlas-app/blob/develop/.eslintrc.json 

Das folgt stärker der NextJS empfehlung die sie gerade geben.

Und das extend dokumentiert NextJS auch so, wie du es verwendet hast… https://nextjs.org/docs/pages/building-your-application/configuring/eslint